connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -43,6 +43,8 @@ message Expression {
     Window window = 11;
     UnresolvedExtractValue unresolved_extract_value = 12;
     UpdateFields update_fields = 13;
+    PythonFunction python_function = 14;
```

Comment:
It looks like this message is never used outside the Python UDF so it should not be listed here.

Reply:
Sounds good.

Reply:
I may want to keep `PythonFunction python_function` in `oneof expr_type` for now since the generated property `python_function` is used as shown below.

```
class PythonUDF(Expression):
    ...
    def to_plan(self, session: "SparkConnectClient") -> proto.Expression:
        ...
        expr.python_udf.function.CopyFrom(self._func.to_plan(session).python_function)
```

Please let me know if there is a better way to get the PythonFunction message.